packages/cli/src/commands/push.js

```diff
@@ -37,7 +37,7 @@ class PushCommand extends Command {
       : path.join(pwd, args.pattern);
 
     if (isFolder(filePattern)) {
-      filePattern = path.join(filePattern, '**/*.{js,jsx,ts,tsx}');
+      filePattern = path.join(filePattern, '**/*.{js,jsx,ts,tsx,html}');
```

Comment:
You also need to update the README and the CLI help text to include `html` as well.

Reply:
done